--- commons-daemon-1.0.2-src/src/native/unix/native/Makefile.in.orig	2010-02-13 17:58:21.000000000 +0100
+++ commons-daemon-1.0.2-src/src/native/unix/native/Makefile.in	2010-05-19 00:10:44.000000000 +0200
@@ -38,7 +38,7 @@
 	$(RANLIB) libservice.a
 
 jsvc: jsvc-unix.o libservice.a
-	$(LDCMD) $(LDFLAGS) jsvc-unix.o libservice.a $(LIBS) -o ../jsvc
+	$(LDCMD) jsvc-unix.o libservice.a $(LIBS) $(LDFLAGS) -o ../jsvc
 
 clean:
 	rm -f $(OBJS) ../jsvc jsvc-unix.o libservice.a
